.gitignore

```diff
@@ -17,3 +17,4 @@ npm-debug.log
 yarn-error.log
 .idea
 .DS_Store
+data.ms/
```

Comment:
This is because I have Meilisearch running by default atm, so this file is being created. Nothing to do with this PR, really!